cdap-kubernetes/pom.xml

```diff
@@ -31,7 +31,7 @@
   <properties>
     <k8s.version>6.0.1</k8s.version>
     <!-- Needs to define the gson and guava versions to override the one from dependency management in cdap pom.xml -->
-    <gson.version>2.6.2</gson.version>
+    <gson.version>2.2.4</gson.version>
```

Comment:
Is this still needed?

Reply:
No, I accidentally changed this when I reverted the other gson change. App-fabric pod failed to start with this change.

It seems like the top-level pom.xml is using gson `2.2.4` but this module is using `2.6.2`...not sure if thats intentional